internal/corazarules/rule_match.go

```diff
@@ -25,6 +23,8 @@ type MatchData struct {
 	Message_ string
 	// Macro expanded logdata
 	Data_ string
+	// Variable
+	Variable_ variables.RuleVariable
```

Comment:
MatchData is one where it is potentially high impact as we could allocate a lot of them per request. However, separating Variable_ from Key_/Value_ seems like a large loss in readability. I would lean towards favoring the readability